operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -23,38 +17,31 @@ The title appears in the left-hand navigation and at the top of the table of con
 
 ### Document sections
 
-To make pages easier for people to quickly scan for the content that they're looking for, break your 
-document up into logical sections. Each section should have a heading, and the heading should relate 
-to the content of the section. A section heading is not required if you have only one section.
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a heading, and the heading should relate to the content of the section. A section heading is not required if you have only one section.
 
 **Use sentence case for the section heading (e.g. "This section is about types of headings") except where a word/phrase is a proper noun (e.g. "Mattermost"), the name of a country/region (e.g. "The United States of America"), and so on.**
 
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+An exception is Customer Education content where this is replaced by Learning Objectives which set clear expectations for a well-defined audience.
```

Comment:
Out of curiosity - if learning objectives set 'clear expectations for a well-defined audience', how much will it differ from the abstract which provides 'a high-level overview of the content .. and outlines the value or goal to the reader', and which 'people read the description to help them decide if the document is the one that they want'

Reply:
Very good find @jasonblais , here are my thoughts:

### Educational Guides - this PR

> clear expectations
- How the reader benefits from reading the guide - what  exact knowledge is taught
- Other expectations
    - How long working through/with the guide will take 
    - What similar topics are not covered
    - The audience is at the center of attention (what can the reader expect)

> for a well-defined audience
 - This audience can be everything from very narrow (System Administrators) to a larger audience (System Administrators, Project Managers, and IT Leaders in general), but it has to be named

### Existing Documentation

> a high-level overview of the content .. and outlines the value or goal to the reader
- What topics the document touches
- *outlines the value or goal* is very universal and unspecific
- the document itself is the center of attention (what does the document bring to the reader)
- reader is not specified in the document itself but by docs category (User, Administrator, Developer)

### TL;DR

- Educational guides have no need to outline the content but set more specific expectations
- different focus in language, focus is the reader as opposed to focus is the document

Reply:
@faase I would recommend adding these examples to the content in this piece.  Even the TL;DR clarifies the differences well.

Reply:
Incorporating this I'd add to the **Documentation Style Guide** section

> When writing educational content the audience is placed at the center of attention, opposing technical documentation where it is the documents content.

and change the **Abstract** section to

> An exception is Customer Education content that has no need to outline the content but set more specific expectations for a well-defined audience. 
>
> Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT Leaders in general). When setting the expectations answer these questions
>
> - How will the reader benefits from reading the guide - what exact knowledge is taught?
> - How long will it take to work through the guide?
> - What similar topics are not covered?

This should cover all the content in this comment but integrate it seemless into the existing structure. WDYT @wiersgallak